tests/e2e/support/objects/account/index.ts

```diff
@@ -1,12 +1,5 @@
 import { Page } from 'playwright'
-
-import {
-  getQuotaValue,
-  getUserInfo,
-  openAccountPage,
-  requestGdprExport,
-  downloadGdprExport
-} from './actions'
+import * as PO from './actions'
```

Comment:
Can we do this in small cases or should there be any reason to have it in upper cases?
```suggestion
import * as po from './actions'
```